pages/training/[slug].tsx

```diff
@@ -25,13 +25,12 @@ const TrainingPage: NextPage<Props> = (props) => {
   const { slug, data } = props;
   const [index, setIndex] = useState(0);
   const [loading, setLoading] = useState(false);
-  const [questionsState, setQuestionsState] = useState<Record<string, any>>({});
-
   const isFinished = index === data.length;
+  const [questionsState, setQuestionsState] = useState<Record<string, any>>(prepareInitialState(data[index]?.options, isMultiple(data[index]?.options)));
```

Comment:
you dont need to pass here the isMultiple result as argument, you have already data[index]?options, and you have the isMultiple function on the utils, so you can use there.